crates/bevy_remote/src/schemas/mod.rs

```diff
@@ -1,4 +1,68 @@
 //! Module with schemas used for various BRP endpoints
 
+use bevy_asset::{ReflectAsset, ReflectHandle};
+use bevy_ecs::{
+    reflect::{ReflectComponent, ReflectResource},
+    resource::Resource,
+};
+use bevy_platform::collections::HashMap;
+use bevy_reflect::{
+    prelude::ReflectDefault, Reflect, ReflectDeserialize, ReflectSerialize, TypeData,
+    TypeRegistration,
+};
+use core::any::TypeId;
+
 pub mod json_schema;
 pub mod open_rpc;
+
+/// Holds mapping of reflect data types to strings,
+/// later on used in Bevy Json Schema.
```

Comment:
What kind of "strings"? Are these the schemas themselves? Type paths? Might be worth clarifying.

Same goes for other mentions of "string" in this file.

Reply:
I don't have idea how to make it more clear ATM. If you have, please suggest 😅